frontend/src/components/modules/routeDetail/AddBuildingPopup.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Dialog from '@mui/material/Dialog';
+import {Building} from '@/api/models';
+
+export interface SimpleDialogProps {
+    open: boolean;
+    buildings: Building[];
+    onClose: (id: number | undefined) => void;
+}
+
+export default function AddBuildingPopup({onClose, buildings, open}: SimpleDialogProps) {
+    return (
+        <Dialog onClose={() => onClose(undefined)} open={open}>
+            <DialogTitle>Voeg een gebouw toe</DialogTitle>
```

Comment:
would be nice to have an Autocomplete component here to be able to search through the buildings and select the one you want. Otherwise scrolling through 100+ buildings would be very annoying.
Adding a user in the planner already uses an Autocomplete like that if want an example

Reply:
Good idea, will look in to it.

Reply:
Issue is created. https://github.com/SELab-2/Dr-Trottoir-3/issues/253 Will add to KANBAN board and assign someone for next week.